core/base/planarGraphLayout/MergeTreeVisu.h

```diff
@@ -0,0 +1,948 @@
+/// \ingroup base
+/// \class MergeTreeVisu
+/// \author Mathieu Pont (mathieu.pont@lip6.fr)
+/// \date 2021.
+///
+/// Visualization module for merge trees.
+///
+
+#ifndef _MERGETREEVISU_H
+#define _MERGETREEVISU_H
+
+#pragma once
+
+#include <FTMTree.h>
+#include <FTMTreeUtils.h>
+
+#include <Debug.h>
```

Comment:
in principle, not necessary.

Reply:
In the next commit: Debug will be removed but FTMTree and FTMTreeUtils are still needed.